en-US/privacy/faq.lang

```diff
@@ -155,6 +156,12 @@ What about my account data?
 We are big believers of data minimization and not asking for things we don't need.
 
 
+## TAG: firefox-accounts-112018
+;You don't need an account to use Firefox. <a href="%(accounts)s"accounts }}">Accounts</a> are required to sync data across devices, but we only ask you for an email address. We don't want to know things like your name, address, birthday and phone number.
+You don't need an account to use Firefox. <a href="%(accounts)s">Accounts</a> are required to sync data across devices, but we only ask you for an email address. We don't want to know things like your name, address, birthday and phone number.
```

Comment:
Have you checked for errors?

Reply:
Gosh, ;-\. However, on my error page, nothing like this shows up, including the ones you mentioned earlier.